packages/transactions/src/coders/transaction.ts

```diff
@@ -669,13 +580,79 @@ export class TransactionBlobCoder extends Coder<TransactionBlob, TransactionBlob
   }
 }
 
+export type TransactionUnknown = BaseTransactionType & {
+  type: TransactionType.Unknown;
+
+  /** Data of the transaction */
+  data?: BytesLike;
+};
+
+export class TransactionUnknownCoder extends Coder<TransactionUnknown, TransactionUnknown> {
+  constructor() {
+    super('TransactionUnknown', 'struct TransactionUnknown', 0);
+  }
+
+  encode(value: TransactionUnknown): Uint8Array {
+    const parts: Uint8Array[] = [];
+
+    parts.push(new NumberCoder('u32', { padToWordSize: true }).encode(value.policyTypes));
+    parts.push(new NumberCoder('u16', { padToWordSize: true }).encode(value.inputsCount));
+    parts.push(new NumberCoder('u16', { padToWordSize: true }).encode(value.outputsCount));
+    parts.push(new NumberCoder('u16', { padToWordSize: true }).encode(value.witnessesCount));
+    parts.push(new PoliciesCoder().encode(value.policies));
+    parts.push(new ArrayCoder(new InputCoder(), value.inputsCount).encode(value.inputs));
+    parts.push(new ArrayCoder(new OutputCoder(), value.outputsCount).encode(value.outputs));
+    parts.push(new ArrayCoder(new WitnessCoder(), value.witnessesCount).encode(value.witnesses));
+
+    return concat(parts);
+  }
+
+  decode(data: Uint8Array, offset: number): [TransactionUnknown, number] {
```

Comment:
Decoding here seems dangerous given we don't know the structure of the bytes.

Reply:
Good point @danielbate , I've removed the encoding/decoding 

6b88425ac